services/static-webserver/client/source/class/osparc/desktop/paymentMethods/PaymentMethods.js

```diff
@@ -101,7 +101,7 @@ qx.Class.define("osparc.desktop.paymentMethods.PaymentMethods", {
         };
         osparc.data.Resources.fetch("paymentMethods", "init", params)
           .then(data => {
-            this.__popUpPaymentGateway(data["paymentMethodId"], data["paymentMethodFormUrl"]);
+            this.__popUpPaymentGateway(data.paymentMethodId, data.paymentMethodFormUrl);
```

Comment:
How is it possible that this changed? Is this an indication that the API changed or was is parsed differently?